src/water_cmd.cpp

```diff
@@ -1361,11 +1361,42 @@ static VehicleEnterTileStatus VehicleEnter_Water(Vehicle *v, TileIndex tile, int
 	return VETSB_CONTINUE;
 }
 
+/**
+ * Check whether a tile with or without a flooding behaviour can be terraformed.
+ *
+ * @param tile Tile to check.
+ * @param z_new The new height the tile would get after being terraformed.
```

Comment:
Obviously the actual code should change too. 😛 

```suggestion
 * @param new_height The new height the tile would get after being terraformed.
```

Reply:
I'm not sure it's a good idea to rename these. The 3 procs already have them named `z_new` and `tileh_new`. Wouldn't it add some confusion?

Reply:
Yes, I didn't realize it was a proc.